regression/cbmc/variable-access-to-constant-array/test.desc

```diff
@@ -1,10 +1,15 @@
-CORE broken-smt-backend
+THOROUGH broken-smt-backend
 main.c
 
 ^EXIT=10$
 ^SIGNAL=0$
 ^VERIFICATION FAILED$
 --
 ^warning: ignoring
+--
 This test actually passes when using the SMT2 back-end, but takes 68 seconds to
 do so.
+
+Field-sensitive encoding of array accesses for an array of 2^16 elements is very
```

Comment:
Given this PR doesn't do array-cell sensitivity yet, this might no longer be relevant

Reply:
Well, it does remain true whether or not we do it right now I'd say.